symplyphysics/laws/astronomy/ratio_of_luminosities_from_ratio_of_masses_of_stars.py

```diff
@@ -23,8 +23,8 @@
 ## L2 - illuminance of second object,
 ## L1 - illuminance of first object.
 
-mass_first = clone_symbol(symbols.basic.mass)
-mass_second = clone_symbol(symbols.basic.mass)
+mass_first = clone_symbol(symbols.mass, display_symbol="m_1")
```

Comment:
Isn't it clunky? Moreover it's inconsistent with alphabetical subscripts.

Reply:
It is more consistent with latex representation, where there is no undescore. As for alphabetical subscripts, we do not have other options to separate symbol and subscript.

Reply:
I'm not sure that Latex is the main thing here. To me it would be more consistent to have, say `q_1 + q_2 = q_total` than `q1 + q2 = q_total`. Plus, the underscore is a clear indication that it corresponds to a subscript in Latex and is not part of the name or something.

Reply:
I tend to use Pythonic style even for physical formulas. In programming we name variables in snake style, i.e. `some_variable` but do not separate numbers with undescores, i.e. `some_variable1`.